lib/components/Tag/Tag.stories.tsx

```diff
@@ -40,3 +40,37 @@ export const Selected: Story = {
     sourceType: 'tyopaikka',
   },
 };
+
+export const Presentation: Story = {
+  parameters: {
+    design: {
+      type: 'figma',
+      url: 'https://www.figma.com/design/6M2LrpSCcB0thlFDaQAI2J/cx_jod_client?node-id=2217-6012&t=AyEPAtDzF6iKjlJw-4',
+    },
+  },
+  args: {
+    label: 'interesting',
+    variant: 'presentation',
+    sourceType: 'kiinnostus',
+  },
+};
+
+export const Multiline: Story = {
+  decorators: [
+    (Story) => (
+      <div style={{ maxWidth: '150px' }}>
+        <Story />
+      </div>
+    ),
+  ],
+  parameters: {
+    design: {
+      type: 'figma',
+      url: 'https://www.figma.com/design/6M2LrpSCcB0thlFDaQAI2J/cx_jod_client?node-id=2217-6095',
```

Comment:
Noted from the multiline example that text is not aligned to left as in the design.

Reply:
Added `ds-text-left` to base classes. Since tags are the same size as their content, it shouldn't effect previous implementation. Looked fine on storybook at least